packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3844,24 +3840,22 @@ export class ContainerRuntime
 		// We choose to close the summarizer after the snapshot cache is updated to avoid
 		// situations which the main client (which is likely to be re-elected as the leader again)
 		// loads the summarizer from cache.
-		if (this.summaryStateUpdateMethod !== "refreshFromSnapshot") {
```

Comment:
Remove the `summaryStateUpdateMethod` member  from container runtime as its not used anymore.

Reply:
Removed